assets/js/directives/only-integer.js

```diff
@@ -0,0 +1,22 @@
+var storefrontApp = angular.module('storefrontApp');
+
+storefrontApp.directive('onlyInteger', function () {
```

Comment:
I doubt about a directive naming.
Maybe smth like vc-available-integers-only?
@asvishnyakov what do you think?

Reply:
i think we had the same directive in odt and meds themes with 'onlyInteger' name

Reply:
Well, if it is a really naming issue, I'd prefer to rename it anyway.

Reply:
No problem. I`ll rename this.